concent_api/core/migrations/0003_add_and_update_deadline_and_size.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-06-19 09:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+from common.helpers import deserialize_message
+from common.helpers import parse_timestamp_to_utc_datetime
+
+
+def populate_subtask_size_field(apps, _schema_editor):
+    Subtask = apps.get_model('core', 'Subtask')
+    StoredMessage = apps.get_model('core', 'StoredMessage')
+    for subtask in Subtask.objects.filter(result_package_size__isnull=True):
+        stored_message = StoredMessage.objects.filter(
+            pk=subtask.report_computed_task_id
+        ).first()
+        if stored_message is not None:
```

Comment:
Shouldn't this migration fail, if `stored_message` is `None`?

Reply:
I think so too. There should be an `assert` or `get()` instead of `filter()`.

See https://github.com/golemfactory/concent/pull/570/files#r198907346

Reply:
Done.